### ISSUE TITLE
Certain emojis glyphs are not rendering properly in VSCode

### ISSUE THREAD

**Issue by weikinhuang** [MonoLisaFont/feedback#145]
## Background information

To help debugging, fill the following information before describing the issue:

* Font edition (i.e. trial, personal, plus, pro): plus
* Font version (i.e. 1.805): 1.808
* The type of the font file you are using (i.e. otf, ttf): otf
* Operating system (i.e. MacOS, Windows): Windows
* Display resolution (i.e. 3072x1920): 1920x1080
* Any other information that might be helpful: Issue started around 1.802 or so

## Issue

I'm having issues with rendering certain emojis when the monolisa font is enabled. Most notably the `✔️` emoji. It renders as a monospaced monochrome glyph instead.

I'm expecting to see ✔️ instead see: 
![image](https://user-images.githubusercontent.com/794658/164534433-1aa27ff3-20a5-4a33-9ff1-e1d91f2a57b3.png)

However emoji glyphs like ❌ and 🗑️ render fine.

font settings in vscode:

```json
{
  "editor.fontSize": 12,
  "editor.fontFamily": "MonoLisa, Consolas, Menlo, Monaco, 'Courier New', monospace",
  "editor.fontLigatures": true,
  "editor.lineHeight": 0,
}
```

**Comment by lovelymono** [comment]
MonoLisa defines some custom emojis like ✔️ and 🔒, to make some commonly used symbols in the CLI fit with the font. There isn't really a way to disable them, as far as I know. You could try putting an emoji font name before `MonoLisa`, like so:

    "editor.fontFamily": "'Apple Color Emoji', MonoLisa, Consolas, ...",

but I'm not sure if it's going to work fine, since (some?) emoji fonts also define common characters like numbers.

**Comment by weikinhuang** [comment]
I've tried that with `Segoe UI Emoji`, but it completely overrides the MonoLisa font, and then I just end up with a weird mess.

**Comment by lovelymono** [comment]
A somewhat simpler option would be to drop the emojis from the main character set and move them into a special set, which could be kept on by default.

**Comment by weikinhuang** [comment]
That would be good, and if anything, can be included separately if needed.

**Comment by okonet** [comment]
Yeah, I think what we should drop emojis on our end. 

So, if there is a symbol like an "checkbox" there are multiple versions of it: https://unicode-table.com/en/sets/check/ I think MonoLisa should not replace Emojis but rather use the "flat" version. So, for checkbox we could move it to `U+2713` and keep `U+2714` intact.

Alternatively, we could move those symbols to a stylistic set. @MarcusSterz this should be simple, isn't it?

WYT?

**Comment by MarcusSterz** [comment]
Yes, @okonet might be a solution. Thinking about how to do it since I can only put a stylistic set to something that is already an existinng glyph. It would be a reverse fallback in a way.

**Comment by okonet** [comment]
It looks like this depends on https://github.com/MonoLisaFont/website/issues/532

**Comment by MarcusSterz** [comment]
I can uncheck them for export as a quick solution. A bit of a shame because emojis’ looks vary from OS to OS and tend to be no design master pieces.

**Comment by MarcusSterz** [comment]
I just found a way to insert emojis in Glyph so we could keep our symbols as stylistic set.

**Comment by MarcusSterz** [comment]
OK, bad news. It is excactly whatI feared: I cannot add the system emojis and make them replaceable (means: I can add them in Glyphs but they don’t show in apps).
So I will exclude them from export generally.

**Comment by bebraw** [comment]
Solved in 2.0. Now we don't support overriding system emojis due to the reasons explained above.
